src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -667,11 +690,42 @@ private void spawnCitizen(CitizenData data)
 
             entity.setPosition(spawnPoint.getX(), spawnPoint.getY(), spawnPoint.getZ());
             world.spawnEntityInWorld(entity);
+            
+            checkAchievements();
 
             markCitizensDirty();
         }
     }
+    
+    /**
+     * Checks if the achievements are valid
+     * 
+     */
+    private void checkAchievements()
+    {
+        // Get the colonies owner UUID
+        final UUID uuidOwner = this.getPermissions().getOwner();
+        // the colonies size
+        final int size = this.citizens.size();
+        
+        EntityPlayer owner = ServerUtils.getPlayerOnServerFromUUID(uuidOwner);
```

Comment:
I'd prefer if we made a getOwner method in Colony that returned the owner EntityPlayer, and then we could call getOwner().triggerAchievement and we could only be getting the owner if we need to.